lib/matplotlib/tests/test_colors.py

```diff
@@ -303,6 +303,9 @@ def test_BoundaryNorm():
     cmref.set_under('white')
     cmshould = mcolors.ListedColormap(['white', 'blue', 'red', 'black'])
 
+    assert_array_equal(cmref.get_over(), mcolors.to_rgba('black'))
```

Comment:
I don't strictly require this, but it could be written more clearly as

```suggestion
    assert mcolors.same_color(cmref.get_over(), 'white')
```

Reply:
Nice! I made that change in ce562f8bd640da5436295ffdd839b7b73887fb3e. I didn't know `same_color` was available. :)